(LoadError) no such file to load -- spring-asciidoctor-backends on apply --recipeName boot-2.7-3.0-upgrade-report2

**Describe the bug**
When I run `apply --recipeName boot-2.7-3.0-upgrade-report2` on a local project, I get an exception instead of migration report.

**To Reproduce**
Steps to reproduce the behavior:
1. `git clone git@github.com:spring-projects-experimental/spring-boot-migrator.git`, currently at https://github.com/spring-projects-experimental/spring-boot-migrator/commit/17e75ac48d6fe3a18387ad389e5d47f8a54893a1
2. `cd spring-boot-migrator`
3. `./mvnw verify -DskipTests`
4. `java -jar applications/spring-shell/target/spring-boot-migrator.jar`
5. `scan ../maintenance-backend`
6. `apply --recipeName boot-2.7-3.0-upgrade-report2`

**Expected behavior**
A migration report.

**Stacktrace**
```
maintenance-backend:> apply --recipeName boot-2.7-3.0-upgrade-report2 
Applying recipe 'boot-2.7-3.0-upgrade-report2'
2022-11-02T16:35:35.071+01:00 [main] WARN FilenoUtil : Native subprocess control requires open access to the JDK IO subsystem
Pass '--add-opens java.base/sun.nio.ch=ALL-UNNAMED --add-opens java.base/java.io=ALL-UNNAMED' to enable.
16:35:36.902 [main] ERROR o.s.boot.SpringApplication - Application run failed
java.util.ServiceConfigurationError: org.asciidoctor.Asciidoctor: Provider org.asciidoctor.jruby.internal.JRubyAsciidoctor could not be instantiated
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:586)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:813)
	at java.base/java.util.ServiceLoader$ProviderImpl.get(ServiceLoader.java:729)
	at java.base/java.util.ServiceLoader$3.next(ServiceLoader.java:1403)
	at org.asciidoctor.Asciidoctor$Factory.create(Asciidoctor.java:528)
	at org.springframework.sbm.boot.upgrade.common.UpgradeReportUtil.writeHtml(UpgradeReportUtil.java:46)
	at org.springframework.sbm.boot.upgrade_27_30.report.SpringBootUpgradeReportRenderer.writeReport(SpringBootUpgradeReportRenderer.java:29)
	at org.springframework.sbm.boot.upgrade_27_30.report.SpringBootUpgradeReportAction.writeReport(SpringBootUpgradeReportAction.java:157)
	at org.springframework.sbm.boot.upgrade_27_30.report.SpringBootUpgradeReportAction.apply(SpringBootUpgradeReportAction.java:134)
	at org.springframework.sbm.boot.upgrade_27_30.report.SpringBootUpgradeReportAction.applyInternal(SpringBootUpgradeReportAction.java:183)
	at org.springframework.sbm.engine.recipe.Action.applyWithStatusEvent(Action.java:37)
	at org.springframework.sbm.engine.recipe.Recipe.apply(Recipe.java:101)
	at org.springframework.sbm.engine.commands.ApplyCommand.execute(ApplyCommand.java:61)
	at org.springframework.sbm.shell.ApplyShellCommand.apply(ApplyShellCommand.java:61)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.shell.command.invocation.InvocableShellMethod.doInvoke(InvocableShellMethod.java:306)
	at org.springframework.shell.command.invocation.InvocableShellMethod.invoke(InvocableShellMethod.java:232)
	at org.springframework.shell.command.CommandExecution$DefaultCommandExecution.evaluate(CommandExecution.java:158)
	at org.springframework.shell.Shell.evaluate(Shell.java:208)
	at org.springframework.shell.Shell.run(Shell.java:140)
	at org.springframework.shell.jline.InteractiveShellRunner.run(InteractiveShellRunner.java:73)
	at org.springframework.shell.DefaultShellApplicationRunner.run(DefaultShellApplicationRunner.java:65)
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:762)
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:752)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1306)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1295)
	at org.springframework.sbm.SpringShellApplication.main(SpringShellApplication.java:27)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:108)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:65)
Caused by: org.jruby.exceptions.LoadError: (LoadError) no such file to load -- spring-asciidoctor-backends
	at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:1017)
	at RUBY.require(uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:85)
	at RUBY.<main>(<script>:1)
```

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04
 - Java: 17.0.4.1-tem (via SDKMan)
 - Version: https://github.com/spring-projects-experimental/spring-boot-migrator/commit/17e75ac48d6fe3a18387ad389e5d47f8a54893a1

**Additional context**
Scanned project is already at Spring Boot 2.7.5 & Java 17; a web application using Spring Data ElasticSearch.

--- a/applications/spring-shell/pom.xml
+++ b/applications/spring-shell/pom.xml
@@ -138,6 +138,10 @@
                                     <groupId>org.asciidoctor</groupId>
                                     <artifactId>asciidoctorj</artifactId>
                                 </dependency>
+                                <dependency>
+                                    <groupId>io.spring.asciidoctor.backends</groupId>
+                                    <artifactId>spring-asciidoctor-backends</artifactId>
+                                </dependency>
                             </requiresUnpack>
                             <mainClass>org.springframework.sbm.SpringShellApplication</mainClass>
                         </configuration>

